horovod/spark/driver/driver_service.py

```diff
@@ -95,21 +98,14 @@ def __init__(self, fn, args, kwargs):
         """Function kwargs."""
 
 
-class DriverService(BasicService):
-    NAME = 'driver service'
-
-    def __init__(self, num_proc, fn, args, kwargs, key):
-        super(DriverService, self).__init__(DriverService.NAME, key)
+class BasicDriverService(BasicService):
```

Comment:
May be better to actually make most of this stuff hosted in horovod.run instead of horovod.spark, since it's not really Spark specific.